conformance/lang/expressions/range-expr/range_expr_exclusive.balt

```diff
@@ -0,0 +1,4979 @@
+Test-Case: output
+Description: Test range expression when values of expressions are positive integers
+Labels: range-expr, iterable-exclusive, int, DecimalNumber, additive-expr
+
+function init() {
+    int startIndex = 0;
+    int endIndex = 6;
+    int count = 0;
+    foreach int i in startIndex ..< endIndex {
+        count = count + 1;
+    }
+    io:println(count); // @output 6
+}
+
+Test-Case: output
+Description: Test range expression when values of expressions are negative integers
+Labels: range-expr, iterable-exclusive, int, DecimalNumber, unary-minus, additive-expr
+
+function init() {
+    int startIndex = -6;
+    int endIndex = -1;
+    int count = 0;
+    foreach int i in startIndex ..< endIndex {
+        count = count + 1;
+    }
+    io:println(count); // @output 5
+}
+
+Test-Case: output
+Description: Test range expression when values of expressions are integers
```

Comment:
Shall we be more specific about the int values instead of saying just integers?

Reply:
Fixed